core/src/styles/_variables.scss

```diff
@@ -2,6 +2,7 @@ $mobileMinWidth: 375px;
 $desktopMinWidth: 600px;
 $desktopMaxWidth: 900px;
 $topNavHeight: var(--luigi__shellbar--height);
+$combinedLayoutGap: 1rem;
```

Comment:
Tested it and the drawer appears where it should be 👍 . 

I am bit confused why specifically **rem** here and why **1** . I am assuming as long as it puts the dialog bog in the right place should be okay, but thought maybe instead of a fixed number we could somehow calculate size/height of breadcrumb and account for it in there 🤔  I am not sure if [ --luigi__breadcrumb--height](https://github.com/SAP/luigi/blob/main/core/src/App.svelte#L2036) can be somehow used or not. 



(Very edge case and probably not used and we don't have to handle it necessarily, but: you can have **Breadcrumbs** **enabled** + **[showAsTabHeader](https://docs.luigi-project.io/docs/navigation-advanced?section=tabnav)= true** and the dialog I think would still need to be adjusted again. 🤔 )

Reply:
It's '1rem' as it combines two layout gaps visible on the screen - it doesn't depend on the breadcrumbs at all:
- top padding value from the main wrapper ('0.5rem')
- gap between header and content elements ('0.5rem')